src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -168,60 +171,63 @@ function WorkspaceWorkflowsPage({policy, betas, route}: WorkspaceWorkflowsPagePr
                     let newReimbursementChoice;
                     if (!isEnabled) {
                         newReimbursementChoice = CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_NO;
-                    } else if (hasVBA && !Policy.isCurrencySupportedForDirectReimbursement(policy?.outputCurrency ?? '')) {
+                    } else if (!!policy?.achAccount && !Policy.isCurrencySupportedForDirectReimbursement(policy?.outputCurrency ?? '')) {
                         newReimbursementChoice = CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_MANUAL;
                     } else {
-                        newReimbursementChoice = hasVBA ? CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_YES : CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_MANUAL;
+                        newReimbursementChoice = CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_YES;
                     }
 
                     const newReimburserEmail = policy?.achAccount?.reimburser ?? policy?.owner;
                     Policy.setWorkspaceReimbursement(policy?.id ?? '', newReimbursementChoice, newReimburserEmail ?? '');
                 },
-                subMenuItems: (
-                    <>
-                        <MenuItem
-                            titleStyle={hasVBA ? styles.textLabelSupportingNormal : styles.textLabelSupportingEmptyValue}
-                            descriptionTextStyle={styles.textNormalThemeText}
-                            title={
-                                hasVBA && policy?.reimbursementChoice === CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_YES
-                                    ? translate('common.bankAccount')
-                                    : translate('workflowsPage.connectBankAccount')
-                            }
-                            description={bankDisplayName}
-                            disabled={isOffline || !isPolicyAdmin}
```

Comment:
Okay, I think we should not have it double-faded so let's definitely solve that

Reply:
> [!note]
> 1. The first disabled grayed-out fade comes from the "**Make or track payments**" toggle when toggled while offline.
> 2. The second one comes from **Connect bank account / Bank account** option which was and  is disabled while offline.
> 3. **Authorized payer** option is enabled while offline -> meaning the **Authorized payer** can be changed optimistically. This is why this one doesn't get double disabled grayed-out fade when "**Make or track payments**" is toggled while offline. 

**ishpaul777**:
> we can add a check !policy?.pendingFields?.reimbursementChoice  && isOffline to solve this

I don't think this logic is good since it would break the logic mentioned at (2.) where **Connect bank account / Bank account** option was and is supposed to be disabled while offline.

**shawnborton**:
> Okay, I think we should not have it double-faded so let's definitely solve that

To do this, instead I would use **Connect bank account / Bank account** option `shouldGreyOutWhenDisabled` to not have the 2nd gray-out fade applied to the option when already applied from "**Make or track payments**" toggle. 

Similarly, we have to do the same for the `OfflineWithFeedback` wrapper of **Authorized payer** option (3.) we add:
```ts
shouldDisableOpacity={isOffline && policy?.pendingFields?.reimbursementChoice && policy?.pendingFields?.reimburser}
```
in order to not double gray-out the **Authorized payer** if already applied from "**Make or track payments**" toggle, when **Authorized payer** is changed while offline. This is how it would look in all scenarios:

1. **Without Bank account added:**

| Offline (can't add / change **Connect bank account** option) |
| ------------- |
| <img width="677" alt="noba-offline" src="https://github.com/Expensify/App/assets/56457735/b4debd17-2dd9-483f-abc2-ac674a36ab09"> |

| Offline (with "Make or track payments" Toggled (optimistically)) |
| ------------- |
| <img width="677" alt="noba-offline-toggled" src="https://github.com/Expensify/App/assets/56457735/462466fd-c516-40d5-852b-85d22686d012"> |

2. **With Bank account added:**

| Offline (can't add / change **Bank account** option) |
| ------------- |
| <img width="580" alt="ba-offline" src="https://github.com/Expensify/App/assets/56457735/38fb1039-4445-4163-820d-de9baeabdd1c"> |

| Offline (with "Make or track payments" Toggled (optimistically)) |
| ------------- |
| <img width="581" alt="ba-offline-toggled-1" src="https://github.com/Expensify/App/assets/56457735/f8f823fa-2fe9-44fa-8156-5f81654dd9f0"> |

| Offline (with "Make or track payments" Toggled + Authorized payer changed (optimistically)) |
| ------------- |
| <img width="696" alt="ba-offline-toggled-2" src="https://github.com/Expensify/App/assets/56457735/43708123-4023-416d-af14-73a3f4bcb9a5"> |

| Offline (with "Make or track payments" Toggled + Authorized payer changed (optimistically) and hover on Authorized payer) |
| ------------- |
| <img width="576" alt="ba-offline-toggled-2-hover" src="https://github.com/Expensify/App/assets/56457735/962986cf-f9df-4bc9-aff5-70ce840a8661"> |

```ts
shouldDisableOpacity={isOffline && policy?.pendingFields?.reimbursementChoice && policy?.pendingFields?.reimburser}
```
^ - Writing it like this is specifically important for the below case, since we don't want to NOT gray out **Authorized payer** when changed offline (optimistically) in case "Make or track payments" was NOT toggled, by only writing it like:
❌ `shouldDisableOpacity={policy?.pendingFields?.reimburser}`
since that would NOT gray out the optimistically set **Authorized payer**, instead we want to gray it out, maintaining the pattern like so:

| Offline (with "Make or track payments" Not Toggled + Authorized payer changed (optimistically)) |
| ------------- |
| <img width="580" alt="ba-offline-non-toggled-changed-payer" src="https://github.com/Expensify/App/assets/56457735/0334cd5f-2412-41bf-8684-54278c2338d0"> |

Please let me know if we're in agreement with the changes mentioned above from a design / code POV.
Once agreed I'll push the changes!

cc @ishpaul777 @shawnborton